traceapp/tmpl/data_rootdir.go

```diff
@@ -0,0 +1,29 @@
+// +build dev
+
+package tmpl
+
+import (
+	"go/build"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+// rootDir is the directory containing the html/template template files.
+var rootDir = filepath.Join(defaultBase("sourcegraph.com/sourcegraph/appdash/traceapp"), "tmpl")
+
+func defaultBase(path string) string {
```

Comment:
I'm not entirely certain, myself. I hadn't looked much into it as it was the original code thet @beyang wrote -- though I did verify that it was indeed working in both `dev` and non-`dev` build modes.